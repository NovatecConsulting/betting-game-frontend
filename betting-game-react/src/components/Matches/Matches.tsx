import React from 'react';
import styles from './Matches.module.css';

const Matches: React.FC = () => (
  <div className={styles.Matches} data-testid="Matches">
    Matches Component
  </div>
);

export default Matches;
