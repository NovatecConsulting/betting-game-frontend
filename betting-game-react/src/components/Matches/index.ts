export { default as Matches } from "./Matches";
