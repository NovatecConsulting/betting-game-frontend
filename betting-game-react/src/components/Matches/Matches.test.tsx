import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Matches from './Matches';

describe('<Matches />', () => {
  test('it should mount', () => {
    render(<Matches />);
    
    const matches = screen.getByTestId('Matches');

    expect(matches).toBeInTheDocument();
  });
});