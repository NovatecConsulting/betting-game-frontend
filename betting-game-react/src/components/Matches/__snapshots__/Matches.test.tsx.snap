// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches component should render and match snapshot with correct content text. 1`] = `
<div
  className="min-h-screen min-w-screen bg-white text-center text-6xl text-black grid items-center tracking-wider"
>
  Matches Component
</div>
`;
